Source/WebKit/NetworkProcess/curl/NetworkProcessMainCurl.cpp

```diff
@@ -35,6 +35,12 @@ class NetworkProcessMainCurl final: public AuxiliaryProcessMainBaseNoSingleton<N
 public:
     void platformFinalize() override
     {
+        Vector<PAL::SessionID> sessionIDs;
+        process().forEachNetworkSession([&sessionIDs](auto& session) {
+            sessionIDs.append(session.sessionID());
+        });
+        for (auto& sessionID : sessionIDs)
+            process().destroySession(sessionID);
```

Comment:
GTK and WPE ports don't use curls, so either split the patch into curl/soup or change the bug title.

Reply:
ok, will create a separate PR and bug report for CURL case